hybridse/src/udf/udf.cc

```diff
@@ -157,6 +157,20 @@ int32_t weekofyear(Date *date) {
     }
 }
 
+void int_to_char(int32_t val, StringRef* output) {
+    val = val % 256;
+    int32_t strl = 1;
+    if (val == 0) {
+        strl = 0;
```

Comment:
I thought this impl is not accurate.
* the output str should always have constant length 1
* for the case `val = 0`, the output string is `\0` with size =1 .  Be aware the difference between C string and this. Here `\0` is not the delimiter.

Reply:
I agree. The result of the expected value is "", and I match the case with "\0", which is not correct. Maybe I'm not quite familiar with the StringRef constructor. In this case, I should revise the expected value to "\0" instead of revising the return value to "". Very appreciated to your idea!